articles/api/management/v2/user-search/v3/index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: User Search
+description: Learn about Auth0's user search query string syntax and how to search for users then sort the results.
+toc: true
+---
+
+# User Search v3
+
+Looking for someone? Auth0's [list or search users](/api/management/v2#!/Users/get_users) endpoint enables you to search for users using [Lucene Query Syntax](http://www.lucenetutorial.com/lucene-query-syntax.html), select the fields to return, and sort the results.
+
+In this article you'll learn about the query string syntax, how to search for users, and how to sort the results.
+
+## Before you start
+
+* If you are using our [user search engine v2](/api/management/v2/user-search/v2), check out the [section on migrating from v2 to v3](#migrate-from-search-engine-v2-to-v3) below.
+* You'll need a token to make requests to the Management API. Check out [the Auth0 Management APIv2 token](/api/management/v2/tokens) for more information.
+* To perform user search requests the `read:users` [scope](/scopes/current) is required.
```

Comment:
remove the `/current` bit from the URL (avoid using versioning in the URLs unless you want to point specific to an outdated version, otherwise versioning will translate /scopes to /scopes/current automatically)